patches/third_party-blink-renderer-core-html-media-autoplay_policy.cc.patch

```diff
@@ -1,19 +1,28 @@
 diff --git a/third_party/blink/renderer/core/html/media/autoplay_policy.cc b/third_party/blink/renderer/core/html/media/autoplay_policy.cc
-index ad5f2bf069270b6ed411e937e762d4d87580a9a7..7dbc643c09c374b9f7db116eb5f309a60da1622f 100644
+index ad5f2bf069270b6ed411e937e762d4d87580a9a7..0b6758efcc1ac6a331150e75ad4ff4ae75ceebce 100644
 --- a/third_party/blink/renderer/core/html/media/autoplay_policy.cc
 +++ b/third_party/blink/renderer/core/html/media/autoplay_policy.cc
-@@ -137,6 +137,10 @@ bool AutoplayPolicy::DocumentHasUserExceptionFlag(const Document& document) {
+@@ -7,6 +7,7 @@
+ #include "build/build_config.h"
+ #include "third_party/blink/public/mojom/autoplay/autoplay.mojom-blink.h"
+ #include "third_party/blink/public/mojom/feature_policy/feature_policy.mojom-blink.h"
++#include "third_party/blink/public/platform/web_content_settings_client.h"
+ #include "third_party/blink/public/platform/web_media_player.h"
+ #include "third_party/blink/public/web/web_local_frame.h"
+ #include "third_party/blink/public/web/web_local_frame_client.h"
+@@ -137,6 +138,11 @@ bool AutoplayPolicy::DocumentHasUserExceptionFlag(const Document& document) {
  // static
  bool AutoplayPolicy::DocumentShouldAutoplayMutedVideos(
      const Document& document) {
 +#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
since we're changing this patch anyway it would be good to convert this to a define `BRAVE_DOCUMENT_AUTOPLAY_MUTED_VIDEOS`